implementation/src/main/java/io/smallrye/config/SecretKeys.java

```diff
@@ -0,0 +1,52 @@
+package io.smallrye.config;
+
+import java.io.Serializable;
+import java.util.function.Supplier;
+
+public class SecretKeys implements Serializable {
```

Comment:
Since this class comprises only static members, it should be `final` and have a `private` constructor.